ci/testkomodo.sh

```diff
@@ -49,8 +49,15 @@ start_tests () {
     export ECL_SKIP_SIGNAL=ON
 
     pushd ${CI_TEST_ROOT}/tests
+
     python -m pytest -n 4 --mpl --benchmark-disable --eclipse-simulator \
         --durations=0 -sv --dist loadgroup
+
+    # Using presence of "bsub" in PATH to detect onprem vs azure
+    which bsub >/dev/null && basetemp=$(mktemp -d -p ~) \
```

Comment:
Should make have it be under `~/pytest_tmp` or something so that we know what these files are.

Reply:
Fixed